### ISSUE TITLE
Magically-colored relations change their color from step to step

### ISSUE THREAD

**Issue by grayswandyr** [AlloyTools/org.alloytools.alloy#230]
Magically-colored relations may change their color from step to step, in a trace, which make their valuation a hard to follow. As magic is the default in the Visualizer for relations, the end result isn't great for beginners in particular. I'd go as far as saying that it's sort of a bug.

**Comment by PATEW** [comment]
+1. As a beginner who relies on magic color, this was a bit confusing to me. glad to know it's on the fix list

**Comment by nmacedo** [comment]
Fixed.
